postgrest/_sync/client.py

```diff
@@ -97,7 +108,13 @@ def rpc(self, func: str, params: dict) -> SyncRPCFilterRequestBuilder[Any]:
             This method now returns a :class:`AsyncFilterRequestBuilder` which allows you to
             filter on the RPC's resultset.
         """
+        method = "POST"
```

Comment:
/nit can consider:

```
method = "HEAD" if head else "GET" if get else "POST"
```

or

```
if head:
    method = "HEAD"
elif get:
    method = "GET"
else:
    method = "POST"
```

as they felt sightly more readable to me though I think it's fine as currently is